lib/helpers/backend/googleBooksSearch.ts

```diff
@@ -0,0 +1,25 @@
+// Note that I was having trouble with https://www.npmjs.com/package/google-books-search
```

Comment:
You can remove this comment now

Reply:
Nice catch lol. Updated in latest commit.